gcp_variant_transforms/libs/preprocess_reporter.py

```diff
@@ -208,7 +208,8 @@ def _generate_conflicting_headers_lines(
   content_lines = []
   for field_id in sorted(conflicts.keys()):
     first_item = True
-    for definition in sorted(conflicts.get(field_id).keys()):
+    for definition in sorted(conflicts.get(field_id).keys(),
+                             key=lambda x: (str(x[0]), x[1])):
```

Comment:
Why we need this change?
This is pretty cryptic, can we make it more clear what's the goal here?

Reply:
Done.